Makefile

```diff
@@ -51,6 +51,8 @@ deploy: assets
 	node ./_deploy/compile-layouts
 	node ./_deploy/s3.js
 	$(MAKE) npm-publish
+	sleep 20
+	nht rebuild --all --serves user-page
```

Comment:
To avoid apps getting out of step we need some way of enforcing this. I figure this is better than slacking the repo owners to update